core/src/main/java/com/linecorp/armeria/server/ServiceConfigSetters.java

```diff
@@ -138,4 +141,10 @@ ServiceConfigSetters blockingTaskExecutor(ScheduledExecutorService blockingTaskE
      * @param numThreads the number of threads in the executor
      */
     ServiceConfigSetters blockingTaskExecutor(int numThreads);
+
+    /**
+     * Defines a custom {@link SuccessFunction} to allow custom definition of successful requests and responses.
+     * {@link MetricCollectingService} and {@link LoggingService} use this custom definition if set.
```

Comment:
```suggestion
     * Sets a {@link SuccessFunction} that determines whether a request was handled
     * successfully or not. If unspecified, {@link SuccessFunction#ofDefault()} is used.
     *
     * @see MetricCollectingServiceBuilder#...
     * @see LoggingServiceBuilder#...
```

Please make the similar change for the client side and any other builders applicable.